### DISCUSSION TITLE
如何将本地html5项目，打包成免安装exe程序，一键启动

### DISCUSSION THREAD

**Discussion by nmww** [tw93/Pake#718] [Q&A]
如何将本地html5项目，打包成免安装exe程序，一键启动

**Comment by Tlntin** [comment]
[wiki](https://github.com/tw93/Pake/wiki/Pake%E6%89%93%E5%8C%85%E9%9D%99%E6%80%81%E6%96%87%E4%BB%B6%E7%A4%BA%E4%BE%8B)

**Reply by Deja-vuuu** [reply]
有内置静态服务器么 ， 如果是一个react项目出来的文件怎么打包呢？
